### DISCUSSION TITLE
Getting started with JavaScript, nodeJs, expressJs and discord bots

### DISCUSSION THREAD

**Discussion by lem0n4id** [WeMakeDevs/classroom-monitor-bot#63] [Q&A] [answered]
I want to contribute to the discord bot but i couldn't find any first timer issue.

I'm not familiar with discord bots and i don't have the skills required. I understand basic syntax of JavaScript since it was my 2nd language after python but i haven't done any projects using JavaScript.

Can anyone link me to resources to help me get started with JavaScript, nodeJs, expressJs and discord bots?

**Comment by Panquesito7** [comment, marked as answer]
There are many many courses out there on how to get started with JavaScript, NodeJS, expressJS, and Discord bots.

This is what I found on Google:

- [Mozilla - JavaScript basics](https://developer.mozilla.org/es/docs/Learn/Getting_started_with_the_web/JavaScript_basics).
- [freeCodeCamp - Node.js and Express.js](https://www.youtube.com/watch?v=Oe421EPjeBE).
- [Discord.js Guide](https://discordjs.guide/)

I hope that helps. 🙂

**Reply by lem0n4id** [reply]
This is helpful. Thanks!
